### PR TITLE
update for rails 5 and ruby 2.4.1

### PR DESCRIPTION
**Pull request by bhfailor**
Made changes to that tests can be run with rails 5.1.4 and ruby 2.4.1.

Commit message should say 'rails 5'.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -144,7 +146,7 @@ DEPENDENCIES
   timecop
 
 RUBY VERSION
-   ruby 2.3.1p112
+   ruby 2.4.1p111
 
 BUNDLED WITH
-   1.12.5
+   1.16.0.pre.2
```
Comment (ramhoj) [inline Gemfile.lock]: Tiny nit-pick: Can we bundle with latest stable bundler version?

**Review by ramhoj — CHANGES_REQUESTED** [review]
Looks great, thank you!

Please resolve the conflicts and have a look at my comment regarding the bundler version and we can merge this :)

**Comment by ramhoj** [comment]
Oh, one more thing. Can you also update the README with the new [Ruby and Rails versions](https://github.com/ramhoj/time-zone-article#ruby-and-rails-version)?

**Comment by bhfailor** [comment]
I think I have responded to your feedback, @ramhoj -
* bundler not prerelease
* update README
* resolved conflicts
* and also updated Travis ruby version

**Comment by ramhoj** [comment]
Looks great @bhfailor, thank you!

**Comment by bhfailor** [comment]
You're welcome, @ramhoj - thanks for sharing this work with the rest of us.

**Comment by ramhoj** [comment]
🙂

**Pull request merged**